zoom/tool.gpt

```diff
@@ -133,19 +135,27 @@ Share Context: Zoom Context
 
 ---
 Name: Get Past Meeting Details
-Description: Get the details of a past Zoom meeting.
+Description: Get the details of a Zoom meeting instance. It is possible that the meeting has not occurred yet.
 Credential: ./credential
 Share Context: Zoom Context
-Param: meeting_id: The ID of the meeting to get the details for
+Param: meeting_id_or_uuid: The ID or UUID of the meeting to get the details for. If providing the meeting ID instead of UUID, the response will be for the latest meeting instance.
 
 #!/usr/bin/env python3 ${GPTSCRIPT_TOOL_DIR}/main.py GetPastMeetingDetails
 
+---
+Name: List Past Meeting Instances
+Description: List all instances of a Zoom meeting.
```

Comment:
Same here.

Reply:
same as above, `instances` indicates it is likely a past meeting instance